typescript/infra/scripts/print-chain-metadatas.ts

```diff
@@ -0,0 +1,17 @@
+import { getArgs, getEnvironmentConfig } from './utils';
+
+// This script exists to print the chain metadata configs for a given environment
+// so they can easily be copied into the Sealevel tooling. :'(
```

Comment:
😢 indeed, but I think these will fit nicely into the CLI. There's already commands to view chains and their addresses (essentially a view into the SDK consts). These can go there too.

Reply:
Ah nice. Think it's worth me keeping this till the CLI gets in?